mne/report.py

```diff
@@ -769,6 +769,9 @@ def _add_figs_to_section(self, figs, captions, section='custom',
             warnings.warn('Could not import mayavi. Trying to render '
                           '`mayavi.core.scene.Scene` figure instances'
                           ' will throw an error.')
+        if np.isscalar(scale) and scale < 1.:
+            raise ValueError('scale, if float, must be >= 1 not %s'
+                             % scale)
```

Comment:
@agramfort better? It's not perfect because e.g. `np.isscalar('a') is True`, but it should work most of the time